test/contracts/apps/disputable/disputable_app.js

```diff
@@ -142,8 +143,12 @@ contract('DisputableApp', ([_, owner, agreement, anotherAgreement, someone]) =>
         await disputable.setAgreement(agreement.address, { from: owner })
       })
 
-      it('does not revert', async () => {
-        await disputable.newAction(0, '0x00', owner)
+      it('submits a new action', async () => {
+        const { receipt } = await disputable.newAction(0, '0x00', owner)
```

Comment:
Looks like some opportunities to use the `EMPTY_BYTES` :)

Reply:
Remember we don't have the helpers here yet haha